packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -443,6 +762,16 @@ class _FloatingActionButtonTransitionState extends State<_FloatingActionButtonTr
   @override
   Widget build(BuildContext context) {
     final List<Widget> children = <Widget>[];
+    if (_currentAnimation.status == AnimationStatus.completed && widget.child != null) { 
+      return new ScaleTransition(
+        scale: widget.fabMotionAnimator.getScaleAnimation(parent: widget.fabMoveAnimation),
```

Comment:
Looks like this scaling animation doesn't update `floatingActionButtonScale` in ScaffoldGeometry.
This should do something similar to what we do below in updateGeometryScale.

Reply:
Done.